design-document.md

```diff
@@ -135,6 +142,37 @@ validity. The aggregators use these proof shares in a protocol designed to
 establish the input's validity, without leaking the input itself. We describe
 this input-validation protocol in detail in {{CITE}}.
 
+### Assembling Reports
+
+As noted above, each client has a collection of measurements that it
+wants to send. Each measurement is characterized by a set of
+parameters that are centrally configured and provided to each client:
+
+- A unique identifier (e.g., "dns-queries-mean")
+- A description of how to collect the measurement (e.g., "count
+  the number of DNS queries")
+- The statistic to be computed over the measurement values (e.g., mean)
+- The rules for what constitutes a valid value (e.g., must be between 0
+  and 10000)
+
+Once the client has collected the measurements to send, it needs to
+turn them into a set of reports. Naively, each measurement would be
+sent in its own report, but it is also possible to have multiple
+measurements in a single report; clients need to be configured with
+the mapping from measurements to reports. The set of measurements
+that go into a report is referred to as the "input" to the report.
+Because each report is independent, for the remainder of this document
+we focus on a single report and its inputs.
+
+The client uses the statistic to be computed in order to know how to
+encode the measurement. For instance, if the statistic is mean, then
+the measurement can be encoded directly. However, if the statistic is
+standard deviation, then the client must send both x and x^2.  Section
+[TODO] describes how to encode measurements for each statistic.
```

Comment:
```suggestion
{{CITE}} describes how to encode measurements for each statistic.
```

Reply:
I think this is actually going to be a cross-reference to some to-be-written section of this doc.

Reply:
Indeed. "{{CITE}}" the way we've been writing this.

Reply:
That's not going to be ideal if/when we want to turn this into an IETF draft, b/c it will cause the formatter to choke.